tests/core/test_check_manifest.py

```diff
@@ -102,13 +106,13 @@ def merge_flow_chip():
     chip = siliconcompiler.Chip('test')
 
     flow = 'test'
-    chip.node(flow, 'import', 'builtin', 'import')
-    chip.node(flow, 'parallel1', 'foo', 'parallel1')
-    chip.node(flow, 'parallel2', 'bar', 'parallel2')
+    chip.node(flow, 'import', siliconcompiler, 'import')
+    chip.node(flow, 'parallel1', foo, 'parallel1')
+    chip.node(flow, 'parallel2', bar, 'parallel2')
```

Comment:
I'm pretty sure we could use "echo" here as well in place of foo/bar/baz in order to avoid creating the empty files